ldms/man/Plugin_lnet_stats.man

```diff
@@ -50,7 +52,6 @@ The default location for the lnet stats path is:
 .SH BUGS
 Assumes file name instead of looking it up from the Lustre runtime libraries,
```

Comment:
When you say assumes, isn't it trying the two different locations? What is the order. Does the the 'file' argument override this?

Reply:
I will try to rephrase the man page details you point out in some satisfactory way.